styles.css

```diff
@@ -117,3 +113,202 @@ body {
 .social-media li {
   list-style: none;
 }
+
+.works {
+  width: 375px;
+  background: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  gap: 88px;
+  position: absolute;
+  height: 2788px;
+}
+
+.cards {
+  width: 327px;
+  padding: 16px 16px 25px 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  height: 552px;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.project1 {
+  margin: auto auto;
+}
+
+.picture {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.project-title {
+  height: 36px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.project-summary {
+  display: flex;
+  align-items: center;
+  gap: 20px;
+  list-style: none;
+}
+
+.back-end-dev {
+  color: #7a869a;
+}
+
+.year {
+  color: #7a869a;
+}
+
+.project-details {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.buttons > * {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  color: #396df2;
+  border: none;
+}
+
+.project-button {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 16px;
+  background: transparent;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  color: #396df2;
+  border-color: #396df2;
+}
+
+.about {
+  background: #fff;
+  order: 2;
+  width: 327px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+.about-detail {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  padding-top: 10px;
+}
+
+.about-connect {
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+  order: 0;
+  text-decoration: none;
+}
+
+.skills {
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+}
+
+.skills li {
+  list-style: none;
+}
+
+.dev-container {
+  width: 303px;
+  height: 66.72px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.dev-tools {
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+  padding-left: 10px;
```

Comment:
Your design looks great. Kindly modify the list of languages to match the design template. The icons are in horizontal alignment with the names which have a background as shown in the illustration.

Your design:
![image](https://user-images.githubusercontent.com/20554583/206013468-cfa54b02-a50e-4968-824e-2b7c7bcdc9c5.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/206012721-c1cfabd6-9b50-4905-84d4-27121876d23d.png)